.editorconfig

```diff
@@ -14,16 +14,14 @@ end_of_line=lf
 trim_trailing_whitespace=true
 insert_final_newline=true
 indent_style=space
-indent_size=4
-
-[*.json]
-indent_style=space
 indent_size=2
+max_line_length = 100
+ij_visual_guides = 100
 
-[{*.ddl,*.sql}]
-indent_style=space
-indent_size=2
-
-[{*.yml,*.yaml}]
-indent_style=space
-indent_size=2
+[*.java]
```

Comment:
I'm confused why these changes are all still here...you may want to rebase and just get rid of some of these commits